### DISCUSSION TITLE
Headless bodies shouldnt really be identifyable

### DISCUSSION THREAD

**Discussion by Rehexr** [FakeFishGames/Barotrauma#10979] [Ideas and suggestions]
Simply put, the fact that a headless body, especially without a card, can still be identified by name, seems a bit odd, and i think that should probably be changed, especially for some more spicy uses of destroying a body to make it unidentifyable.

**Comment by MarkCringan** [comment]
Assuming they don't have an ID card, right?

**Comment by Rehexr** [comment]
Forgot to mention it, but yes
